src/superphot_plus/fit_numpyro.py

```diff
@@ -577,6 +580,8 @@ def numpyro_single_curve(lc, output_dir=FITS_DIR, sampler="svi"):
         Directory to save outputs to. Defaults to FITS_DIR.
     sampler : str
         The MCMC sampler to use. Defaults to "svi".
+    telescope : Telescope
```

Comment:
Parameter in comment (`telescope`) does not match the new function parameter `priors`.